proxies/pdl-proxy/src/main/java/no/nav/testnav/proxies/pdlproxy/PdlProxyApplicationStarter.java

```diff
@@ -54,7 +54,7 @@ public RouteLocator customRouteLocator(RouteLocatorBuilder builder, StsOidcToken
         return builder
                 .routes()
                 .route(createRoute("pdl-api", "https://pdl-api.dev.adeo.no", addAuthorizationAndNavConsumerTokenToRouteFilter))
-                .route(createRoute("pdl-testdata", "https://pdl-testdata.dev.adeo.no", addAuthenticationHeaderFilter))
+                .route(createRoute("pdl-testdata", "https://pdl-testdata.dev.intern.nav.no", addAuthenticationHeaderFilter))
```

Comment:
Mener du gjorde en oppdatering til å bruke service discovery syntaks, hvor er det blitt av den?

Reply:
Ikke på pdl-testdata, der var vel noe gcp->gcp spesifikke greier mener jeg? 😅  Men siden backend og pdl-testdata er på fss begge to så kan det kanskje endres?

Reply:
Det er noe annet, men ...
Husker jeg feil at du byttet overnevnte pdl-testdata til å gå mot 
http://pdl-testdata.pdl.svc.nais.local
?

Reply:
Hmm, det har ikke jeg gjort tror jeg 😅  Skal vi endre til å bruke det isteden da? 😄

Reply:
Ja vi kan ta det når du kommer hit.